sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -4500,6 +4499,23 @@ public void testIsNullTrueFalse() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  @Test
+  public void testZetaSQLBitOr() {
```

Comment:
Nice! Surprisingly you can figure out how to enable this function in ZetaSQL dialect!

Reply:
Zeta was surprisingly easier than Calcite!